pkg/types/health.go

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package types
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/gardener/etcd-backup-restore/pkg/wrappers"
+	flag "github.com/spf13/pflag"
+)
+
+const (
+	// DefaultEnableEtcdLeaseRenewal is a flag where a user can choose to turn off this lease renewal feature
+	DefaultEnableEtcdLeaseRenewal = false
+	// FullSnapshotLeaseName is the name for the delta snapshot lease.
+	FullSnapshotLeaseName = "full-snapshot-revisions"
+	// DeltaSnapshotLeaseName is the name for the delta snapshot lease.
+	DeltaSnapshotLeaseName = "delta-snapshot-revisions"
+	// DefaultHeartbeatDuration is the default heartbeat duration or lease renewal deletion.
+	DefaultHeartbeatDuration = 30 * time.Second
+	// DefaultMemberGarbagecollectionSchedule is the default member garbage collection period.
+	DefaultMemberGarbagecollectionSchedule = 60 * time.Second
```

Comment:
Post our recent discussion, you will be removing the membergarbagecollection package for now and re-introducing it later right? But in any case, please rename this variable to `DefaultMemberGarbageCollectionPeriod`, since it defines a period, not a cron schedule as such.